### ISSUE TITLE
Capture is not working on Android 5.0.1 (Solved)

### ISSUE THREAD

**Issue by qvillery** [Neraud/PADListener#112]
I've tried using Manual set up with localhost as the proxy and 8008 as the port, listener will start and I will start PAD then listener will hang and not capture any data.

I've also tried with the Auto start, capture & sync and received the same results.

My phone is an LG G3 running Lollipop 5.0.1, I can provide any additional information as needed.

**Comment by Quarks0** [comment]
Try manual mode again, however make sure you're logged in to PAD before starting the listener, then once the listener is started log back out then back in.

**Comment by qvillery** [comment]
No difference on using manual again. Do you think it might be something with my proxy settings?

**Comment by Quarks0** [comment]
Here's a video I made for a couple of users on reddit a bit ago - give it a quick watch and let me know if  you still run into issues when you do the same thing https://www.youtube.com/watch?v=zJ-DhB4Ay1w

**Comment by qvillery** [comment]
Yeah I'm still getting issues. When I get back into PAD after going to the title screen I don't get a toast nor does the capture start.

**Comment by Quarks0** [comment]
Uh mind doing a screen capture or something? If there's no notification that the proxy has started then something's not going right in the set up

**Comment by qvillery** [comment]
Here's a imgur link since I don't have any capture programs. If you need more I'll see what I can put together. http://imgur.com/a/77Z4T Thanks so much for trying to help :)

**Comment by Quarks0** [comment]
Hmm I'm assuming you're changing wifi settings from within padlistener (if not try that), not from the drop down menu. Otherwise, it does seem to look ok. And you're not at the login screen when you switch to PAD, correct? You have to be logged in when the listener starts, then log out and back in

**Comment by qvillery** [comment]
Correct, I change it through padlistener and I'm logged in on PAD then I go back to title screen and start.

**Comment by Quarks0** [comment]
Hm ok. Let's take a step back - while the proxy is on, can you access websites normally?

**Comment by qvillery** [comment]
Yes

**Comment by Quarks0** [comment]
Ah ok. Then for some reason your proxy settings aren't being saved. When the proxy is active, you should not be able to connect to sites (chrome should say something along the line of err_proxy_failed). Let me look into that now.

Edit: In the meantime, try changing your wifi settings outside padlistener and give it a go?

Edit2: Are you putting your wifi password when you change the proxy settings? My ROM won't let me save until I put in my router's password but apparently other ROMs allow you to hit the save button (but not actually save) as long as the proxy address and port are there.

**Comment by qvillery** [comment]
Hey, I got it working!I restarted my phone and then I reset the proxy (set it to none, saved, set to manual) reopened padlistener, then PAD, and it worked. I really appreciate all your help :)

**Comment by Quarks0** [comment]
Haha maaan that is janky. Good to hear everything's working now =)

**Comment by qvillery** [comment]
I have no clue why it worked this time, but that's computer science/computers in a nutshell.

**Comment by sickcat99** [comment]
I am getting this exact same issue. I've tried everything here and still not working. 
When using manual I tried using other websites and the Web is blocked so proxy is working.
